gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaJobStatusMonitor.java

```diff
@@ -409,4 +428,7 @@ public static long getExecutionIdFromTableName(String tableName) {
 
   protected abstract org.apache.gobblin.configuration.State parseJobStatus(GobblinTrackingEvent event);
 
+  public static boolean isExceptionInstanceOf(Exception exception, List<Class<? extends Exception>> typesList) {
+    return typesList.stream().anyMatch(e -> e.isInstance(exception));
+  }
```

Comment:
NBD, but this would better belong in a util lib

Reply:
yes, just could not find a relevant util class !